### ISSUE TITLE
[Change request] Docker-compose without InfluxDB and Grafana

### ISSUE THREAD

**Issue by corasaniti** [stefanwalther/speedy#15]
It would be nice if you can provide a docker-compose file with only speedy service, for already installation of InfluxDB and Grafana.
you could provide a configuration file to specify host and port of existing InfluxDV and Grafana installations
Thank you
Piero

**Comment by jiripech** [comment]
You can easily do this by specifying where speedy docker container finds the InfluxDB. If you already have InfluxDB and Grafana, all you have to do is configuring speedy via ENV:

```bash
SPEEDY_DB_HOST="your_influxdb_host"
SPEEDY_DB_POST="your_influxdb_port"
INFLUXDB_DB="your_influxdb_database_name"
```

**Comment by corasaniti** [comment]
Ok so just docker run with the ENV properties

SPEEDY_DB_HOST="your_influxdb_host"
SPEEDY_DB_POST="your_influxdb_port"
INFLUXDB_DB="your_influxdb_database_name"


Also can you provide a json export of the Grafana dashboard so that I can change the datasource reference?
Thanks
Piero

**Comment by jiripech** [comment]
Yes, you are right. I'm actually not using this version of speedy, but I can probably help you with the datasource configuration:

![image](https://user-images.githubusercontent.com/3009963/113338650-ac625600-9329-11eb-9c04-79153e5fc4df.png)

Does it work for you? If so, you can try using [this file](https://github.com/stefanwalther/speedy/files/6245785/Internet.Connection.Speed-1617301900253.json.zip) from my Grafana, but I'm using a newer version of speedy from [here](https://hub.docker.com/repository/docker/hqcz/speedy) which also logs latency, so I'm using it as `ping` on the dashboard.

**Comment by corasaniti** [comment]
everything works perfectly thanks :-)
It would be nice to have a build for the ARM architecture so that you can use it with the Raspberry :-)


Thanks
